mock/py/mockbuild/util.py

```diff
@@ -1277,17 +1317,18 @@ def load_config(config_path, name, uidManager, version, pkg_python_dir):
     config_opts['config_file'] = chroot_cfg_path
 
     cfg = os.path.join(config_path, 'site-defaults.cfg')
-    do_update_config(log, config_opts, cfg, uidManager, name)
-
-    do_update_config(log, config_opts, chroot_cfg_path, uidManager, name, skipError=False)
+    do_update_config_paths(log, config_opts, cfg, name)
```

Comment:
Yes, because we do only 'list.append()' operation in function do_update_config_paths().

So the logic is to grab all config path, then grab all content of all paths as one, then do template rendering and do exec() of result to evaluate config